### PR TITLE
chore(go.mod): extract core workflows replaces

### PR DESCRIPTION
**Pull request by janisz**
## Description

Assign replacements to @stackrox/core-workflows  

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

### Here I tell how I validated my change

`go mod tidy` 

### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.

### CODE REVIEW

**Comment by janisz** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/10408?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#10409** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/10409?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#10408** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/10408?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#10407** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/10407?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#10373** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/10373?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 1 other dependent PR ([#10390](https://github.com/stackrox/stackrox/pull/10390) <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/10390?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#10375** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/10375?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @janisz and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: go.mod [hunk 1]
```diff
@@ -445,13 +445,13 @@ require (
 //
 // The `go mod tidy` takes care of normalizing the symbol version information (e.g. branch name) which is required
 // for Go build tools to accept the `go.mod`.
+// github.com/stackrox/helm-operator is a modified fork of github.com/operator-framework/helm-operator-plugins that
```
Comment (rukletsov) [inline go.mod:448]:
```suggestion

// github.com/stackrox/helm-operator is a modified fork of github.com/operator-framework/helm-operator-plugins that
```

**Review by rukletsov — APPROVED** [review]

**Comment by janisz** [comment]
### Merge activity

* **Mar 19, 6:01 PM GMT+1**: [Graphite](https://app.graphite.dev/github/pr/stackrox/stackrox/10408) rebased this pull request after merging its parent, because this pull request is set to merge when ready.

**Pull request merged**